usr/local/share/bastille/bootstrap.sh

```diff
@@ -164,6 +164,7 @@ bootstrap_directories() {
         else
             mkdir -p "${bastille_templatesdir}"
         fi
+        ln -s "${bastille_sharedir}/templates/default" "${bastille_templatesdir}/default"
```

Comment:
This else will only be hit on new deployments. There needs to be an additional check for existing deployments, otherwise you'll get an error when creating new jails.

```
[thin1]:
thin1: created

default/thick not found.
[thin1]:
thin1: removed
```

Reply:
Nice catch. I added a check in `create.sh` for this.